src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -62,7 +62,10 @@ bool Flag_CoreCollapse( const int i, const int j, const int k, const int lv, con
    }
 
 
-// (2) always refined to highest level in the region with r < 30 km, if allowed
+// (2) always refined to highest level in the region with r < 30 km
+   if (  !MaxRefine  &&  ( r < amr->dh[lv] )  &&  ( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )  )
```

Comment:
- I prefer to define a global static variable (e.g. `Half_SQRT3`), and replace `sqrt(0.75)` by it to save time.
- I am wondering whether `( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )` is necessary. The condition ` r < amr->dh[lv] ` is always satisfied for the innermost cells on any level (0.5 * dh * sqrt(3) ~ 0.8 * dh < dh), and thus the innermost region will be refined to allowed highest level.

Reply:
I think you are right, we don't need the condition `( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )`. So I just removed this condition but left `r < amr->dh[lv]`